p2p/constants.go

```diff
@@ -41,3 +41,12 @@ const Direct BroadcastMethod = "Direct"
 
 // Broadcast defines a broadcast message
 const Broadcast BroadcastMethod = "Broadcast"
+
+// Network defines the network a message belongs to
+type Network string
```

Comment:
I do not think this lib should know about how it will be used.
Suggestion to refactor and change the `MessageProcessor` interface to include the Messenger that triggered the call. Something like
```GO
ProcessReceivedMessage(message MessageP2P, fromConnectedPeer core.PeerID, source Messenger) error
```